src/JuMP.jl

```diff
@@ -470,6 +468,30 @@ function unset_silent(model::Model)
     return MOI.set(model, MOI.Silent(), false)
 end
 
+"""
+    writeLP(model::Model, fname::AbstractString)
+
+Exports the input model as a LP file. To compress the output file, use a `.gz` extension.
+The MPS format is equation-oriented, and is adapted for human consumption.
+"""
+function writeLP(model::Model, fname::AbstractString)
+    lp_model = MathOptFormat.LP.Model()
+    MOI.copy_to(lp_model, backend(model))
+    MOI.write_to_file(lp_model, fname)
```

Comment:
How do the JuMP variable names translate to variable names in the LP file? What happens if multiple variables have the same (or empty) names? What if the variable name is incompatible with LP format, e.g., "5 * x2 + x3"?

Reply:
Open issue: https://github.com/odow/MathOptFormat.jl/issues/67

There is code in LPWriter.jl to sanitize the names, but I haven't ported it yet.